go/interpreter/lfvm/instructions.go

```diff
@@ -1044,10 +1044,7 @@ func neededMemorySize(c *context, offset, size *uint256.Int) (uint64, error) {
 }
 
 func genericCall(c *context, kind tosca.CallKind) {
-	warmAccess, coldCost, err := addressInAccessList(c)
-	if err != nil {
-		return
-	}
+
```

Comment:
can you explain me where is the access list checked now?  why is it gone ?

Reply:
The return of `isWarm := c.context.AccessAccount(toAddr)` is what checks the access now. since `IsAddressInAccessList` is deprecated.